### PR TITLE
feat(forum-theme): update footer FEB23

### PR DESCRIPTION
**Pull request by ahmadabdolsaheb**
Checklist:

- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ ] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: settings.yml [hunk 1]
```diff
@@ -39,66 +39,66 @@ footer_donation_cta:
...
+article8link: "https://www.freecodecamp.org/news/javascript-convert-string-to-number-js-string-to-int-example/"
+article9title: "Python Global Variable"
+article9link: "https://www.freecodecamp.org/news/global-variable-in-python-non-local-python-variables/"
+article10title: "Python Set"
+article10link: "https://www.freecodecamp.org/news/python-set-how-to-create-sets-in-python/"
+article11title: "What is REST?"
+article11link: "https://www.freecodecamp.org/news/what-is-rest-rest-api-definition-for-beginners/"
+article12title: "What is CRUD?"
+article12link: "https://www.freecodecamp.org/news/crud-operations-explained/"
+article13title: ""%.2f in Python""
```
Comment (ahmadabdolsaheb) [inline settings.yml:68]:
```suggestion
article13title: "%.2f in Python"
```

File: settings.yml [hunk 2]
```diff
@@ -39,66 +39,66 @@ footer_donation_cta:
...
+article13link: "https://www.freecodecamp.org/news/2f-in-python-what-does-it-mean/"
+article14title: "Git Reset Origin"
+article14link: "https://www.freecodecamp.org/news/git-reset-origin-how-to-reset-a-local-branch-to-remote-tracking-branch/"
+article15title: "What is Localhost?"
+article15link: "https://www.freecodecamp.org/news/what-is-localhost/"
+article16title: "JS Array.includes()"
+article16link: "https://www.freecodecamp.org/news/"check-if-an-item-is-in-an-array-in-javascript-js-contains-with-array-includes/
```
Comment (ahmadabdolsaheb) [inline settings.yml:75]:
```suggestion
article16link: "https://www.freecodecamp.org/news/check-if-an-item-is-in-an-array-in-javascript-js-contains-with-array-includes/"
```

**Review by naomi-lgbt — APPROVED** [review]

File: settings.yml [hunk 3]
```diff
@@ -39,66 +39,66 @@ footer_donation_cta:
...
+article20link: "https://www.freecodecamp.org/news/how-to-create-a-set-in-python/"
+article21title: "Remove a Dict in Linux"
+article21link: "https://www.freecodecamp.org/news/how-to-remove-a-directory-in-linux/"
+article22title: "What is Coding Used For?"
+article22link: "https://www.freecodecamp.org/news/what-is-coding-used-for/"
+article23title: "JavaScript String Format"
+article23link: "https://www.freecodecamp.org/news/javascript-string-format-how-to-format-strings-in-js/"
+article24title: "JavaScript String to Date"
+article24link: "https://www.freecodecamp.org/news/javascript-string-to-date-date-parsing-in-js/"
+article25title: "What’s a Data Analyst Do?"
```
Comment (ShaunSHamilton) [inline settings.yml:92]: Same as the other one. Are we ok with special apostrophes?

```suggestion
article25title: "What's a Data Analyst Do?"
```

File: settings.yml [hunk 4]
```diff
@@ -39,66 +39,66 @@ footer_donation_cta:
...
+article16link: "https://www.freecodecamp.org/news/check-if-an-item-is-in-an-array-in-javascript-js-contains-with-array-includes/"
+article17title: "Compare Dates in JS"
+article17link: "https://www.freecodecamp.org/news/javascript-date-comparison-how-to-compare-dates-in-js/"
+article18title: "Python Split String"
+article18link: "https://www.freecodecamp.org/news/how-to-split-a-string-in-python/"
+article19title: "Python .enumerate()"
+article19link: "https://www.freecodecamp.org/news/python-enumerate-what-is-the-enumerate-function-in-python/"
+article20title: "Create a Set in Python"
+article20link: "https://www.freecodecamp.org/news/how-to-create-a-set-in-python/"
+article21title: "Remove a Dict in Linux"
```
Comment (ShaunSHamilton) [inline settings.yml:84]:
```suggestion
article21title: "Remove a Dir in Linux"
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
As pointed out in the other PR

File: settings.yml [hunk 5]
```diff
@@ -39,66 +39,66 @@ footer_donation_cta:
...
+article14link: "https://www.freecodecamp.org/news/git-reset-origin-how-to-reset-a-local-branch-to-remote-tracking-branch/"
+article15title: "What is Localhost?"
+article15link: "https://www.freecodecamp.org/news/what-is-localhost/"
+article16title: "JS Array.includes()"
+article16link: "https://www.freecodecamp.org/news/check-if-an-item-is-in-an-array-in-javascript-js-contains-with-array-includes/"
+article17title: "Compare Dates in JS"
+article17link: "https://www.freecodecamp.org/news/javascript-date-comparison-how-to-compare-dates-in-js/"
+article18title: "Python Split String"
+article18link: "https://www.freecodecamp.org/news/how-to-split-a-string-in-python/"
+article19title: "Python .enumerate()"
```
Comment (ahmadabdolsaheb) [inline settings.yml:80]:
```suggestion
article19title: "Python enumerate()"
```

**Review by ShaunSHamilton — APPROVED** [review]

**Pull request merged**